draft-bbs-signatures.md

```diff
@@ -59,15 +59,13 @@ BBS is a digital signature scheme categorized as a form of short group signature
 
-However, traditional digital signature schemes require both the signature and the entire message to be disclosed during verification, constraining its usage in certain applications.
+Beyond the core properties of a digital signature scheme, BBS signatures provide multiple additional unique properties, three key ones are:
 
-The BBS Signature scheme provides multiple unique properties when compared with conventional digital signature schemes, however three key ones are:
+**Selective Disclosure** - The scheme allows a signer to sign multiple payloads and produce a single output signature. An intermediary (prover) then possessing the payloads and the signature can generate a proof whereby they can choose which payloads to reveal.
 
-**Selective Disclosure** - The scheme allows a signer to sign multiple payloads while still produce a single output signature. An intermediary then possessing the payloads and the signature can generate a proof whereby they can choose which payloads to reveal.
+**Unlinkable Proof Presentations** - The proofs generated by the scheme are known as a signature proof of knowledge, meaning a verifying party in receipt of a proof is unable to determine which signature was used to generate the proof, removing a common source of correlation. In general each proof generated is indistinguishable from random even for two proofs generated from the same signature.
 
-**Unlinkable Proof Presentations** - The proofs generated by the scheme are known as a signature proof of knowledge meaning a verifying party in receipt of a proof is unable to determine which signature was used to generate the proof.
-
-**Scoped Proofs** - The proofs generated by the scheme feature a presentation header where the proving party is able to include any information they would like to cryptographically tie to the proof. This can include a cryptographic nonce and or an audience/domain identifier to ensure the generated proof can only be used appropriately and provides a way for a verifying/relying party to detect a replay attack.
+**Proof of Posession** - The proofs generated by the scheme prove to a verifier that the party who generated the proof (prover) was in posession of a signature without revealing it. Because the scheme also supports binding a presentation header to the generated proof it makes the scheme suitable as a proof of posession protocol. Note the presentation header can include abitrary information such as a cryptographic nonce and or an audience/domain identifier to ensure the generated proof can only be used appropriately, including providing a way for a verifier to detect a replay attack.
```

Comment:
*possession :)

Reply:
```suggestion
**Proof of Possession** - The proofs generated by the scheme prove to a verifier that the party who generated the proof (prover) was in possession of a signature without revealing it. Because the scheme also supports binding a presentation header to the generated proof it makes the scheme suitable as a proof of possession protocol. Note the presentation header can include arbitrary information such as a cryptographic nonce and or an audience/domain identifier to ensure the generated proof can only be used appropriately, including providing a way for a verifier to detect a replay attack.
```